lib/components/settings/widgets/settings_card.dart

```diff
@@ -18,12 +18,11 @@ import 'package:flutter/material.dart';
 import 'dart:math' as math;
 import 'package:pangolin/components/settings/models/settings_element_model.dart';
 
-// ignore: must_be_immutable
 class SettingsCard extends SettingsElementModel {
-  Widget? content, trailing, leading;
-  String? title, subtitle;
-  ValueChanged<bool>? onToggle;
-  VoidCallback? onTap;
+  final Widget? content, trailing, leading;
```

Comment:
just a small nit as discussed in the discord server, consider declaring each var in its own line for better readability